### PR TITLE
improve pico integration docs for oci sdk

### PR DESCRIPTION
**Pull request by trentjeff**
This is a follow-up to some of the comments @ljnelson provided on https://github.com/helidon-io/helidon/pull/6982

### CODE REVIEW

**Comment by trentjeff** [comment]
@romain-grecourt @tomas-langer @m0mus - i happened to notice https://github.com/helidon-io/helidon/pull/6969.  Seems like our nima integration examples should also be updated to include pico and injection - thoughts on this?

File: integrations/oci/sdk/README.md
Comment (ljnelson) [inline integrations/oci/sdk/README.md:14]: (My earlier "module" comments apply here)

File: integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java
Comment (ljnelson) [inline integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java:281]: "or use" -> "for use", or just remove the phrase since it adds nothing

File: integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java
Comment (ljnelson) [inline integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java:285]: no need for "for use"; adds nothing. Maybe just "the list of applicable authentication strategies". Or are they strategy descriptors (the term used in the terminology you pointed the users at originally)?

**Review by ljnelson — APPROVED** [review]
I've left further suggestions but clicked the big approve button in case you need to get this in as-is.

**Comment by tomas-langer** [comment]
This will be in conflict with #7008, please do not merge until that PR is merged. Thanks

File: integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java
Comment (trentjeff) [reply integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java]: pushed

File: integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java
Comment (trentjeff) [reply integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciConfigBean.java]: pushed

File: integrations/oci/sdk/README.md
Comment (trentjeff) [reply integrations/oci/sdk/README.md:14]: Discussed this with Tomas and he suggests it can be left as just "module".

File: integrations/oci/sdk/README.md
Comment (trentjeff) [reply integrations/oci/sdk/README.md]: pushed

File: integrations/oci/sdk/README.md
Comment (trentjeff) [reply integrations/oci/sdk/README.md:10]: Discussed this with Tomas and he suggests it can be left as just "module".

**Comment by m0mus** [comment]
This document covers OCI integration with SE. You should mention it only once and not include information about CDI modules.

**Comment by m0mus** [comment]
We have not agreed on naming. You will need to fix it later. I am suggesting filing a follow-up issue.

**Comment by m0mus** [comment]
Fix (remove) indent in snippets.

**Comment by m0mus** [comment]
It makes sense to indicate exactly the place of snippets in the pom file. Something like: adds the following code under `build\pluginManagement\plugin` or similar.

**Comment by m0mus** [comment]
It would be good to know what modules you can inject and the names of the classes. I expect questions like: I would like to use a component XXX, how can I inject it? Documentation should answer it.

**Comment by trentjeff** [comment]
Filed follow-up using https://github.com/helidon-io/helidon/issues/7076

**Review by tomas-langer — APPROVED** [review]

**Pull request merged**
